docs/manual/philosophy.md

```diff
@@ -0,0 +1,149 @@
+---
+layout: default
+title: rpm.org - RPM's Philosophy
+---
+# RPM's Philosophy
+
+The RPM package manager is a general purpose software manager. It
```

Comment:
I think a brief definition of what that means would be in order here, something that covers the two sides of our operation: wrapping software into packages that the other side knows how to manipulate: install, update, remove, verify.

Reply:
I'd also place a summary of the founding design principles right here in the beginning to set a frame of mind:
- unattended operation
- reproducibility
- verifiability